consensus/src/user/provisioners.rs

```diff
@@ -17,12 +17,19 @@ use std::mem;
 use super::committee::Committee;
 
 pub const DUSK: u64 = 1_000_000_000;
+const MINIMUM_STAKE: u64 = 1_000 * DUSK;
```

Comment:
I think we wanna bump this up to 5_000 DUSK

Reply:
I'll create a different PR for that, so to not mix rebase with new features